AutoDarkModeApp/Pages/PageAbout.xaml

```diff
@@ -73,7 +73,7 @@
 
             <TextBlock 
                 Margin="0,0,0,0"
-                Text="Version" 
+                Text="{x:Static p:Resources.AboutTextBlockVersion}" 
```

Comment:
The localization on the about page is omitted intentionally, as it is debug information meant for the developers.
Might seem pendantic, but this specific entry should remain as "Version"